src/fix_langevin.h

```diff
@@ -139,6 +140,18 @@ E: Fix_modify temperature ID does not compute temperature
 
 The compute ID assigned to the fix must compute temperature.
 
+E: Fix langevin gjf cannot have period equal to dt/2
```

Comment:
If there are new errors, they should also be added to `doc/src/Errors_messages.txt`

Reply:
I have added the new errors and warnings.